sugar/lib/src/time/zone/providers/embedded/tzdb.dart

```diff
@@ -0,0 +1,245 @@
+import 'package:meta/meta.dart';
+import 'package:sugar/src/time/zone/providers/embedded/embedded_timezone.dart';
+import 'package:b/b.dart';
```

Comment:
what is b package?

Reply:
It's used for decoding custom encoding.

The https://github.com/kshetline/tubular_time_tzdb packages uses a clever form of base60 (e.g. `2ldXH`)for encoding timezone values. It is not a standard encoding. The `b` package makes it really easy to define a simple encoding.

Reply:
https://pub.dev/packages/b

Reply:
@Pante Are you ok with this?

Reply:
Yup!